interface/src/app/sample/sampleMocks.spec.js

```diff
@@ -0,0 +1,97 @@
+angular.module('adage.mocks.sample', [])
```

Comment:
The files named `*.spec.js` are designated specifically for unit tests, yes. The mock data are designated specifically for the same tests, so I'm not sure why it would be a problem to name them this way.

From a practical perspective, I can't name them anything else without modifying the naming system used by the build system. We don't want mock data built and compiled into our deployed system, and that is what would happen if I named them `somethingelse.js` without the `spec.js` ending. Have a look at `build.config.js`, where the `app_files` are defined starting at line 22. Notice that the `*.spec.js` files are explicitly *excluded* from the `js` definition and *included* only in the `jaunt` definition. This is exactly what we want.

Thank you for pointing out the potential problem with dependencies. I've thought about that carefully as well. Now that we have refactored to remove the conflict with two `Sample` services, `sample_annotation/annotation.js` already depends upon having `sample/services.js` injected. I think it is reasonable for the unit tests to follow that same path of dependency, so it is logical that `annotation.spec.js` should depend upon some testing infrastructure that is also required by `sample/services.spec.js`. Remember that I'm doing this to uphold the DRY principle that `codeclimate` is helping to enforce. This structure helps keep the testing data for this service right along with the service itself, where it's bound to be needed again.

If this still doesn't feel right to you, please create an issue so we can discuss it further. We need this patch to get merged so it doesn't hold up the next round of pull requests.